doc/security-warnings.md

```diff
@@ -0,0 +1,34 @@
+Security Warnings
+====================
+
+Security Audit
+--------------
+
+Zcash has not yet been subjected to a formal third-party security review. This
+section will be updated with links to security audit reports in the future.
+
+x86-64 Linux Only
+-----------------------
+
+There are [known bugs](https://github.com/scipr-lab/libsnark/issues/26) which
+make proving keys generated on 64-bit systems unusable on 32-bit and big-endian
+systems. It's unclear if a warning will be issued in this case, or if the
+proving system will be silently compromised.
```

Comment:
[src/common/rng.tcc](https://github.com/scipr-lab/libsnark/blob/master/src/common/rng.tcc)#L21 has
```
template<typename FieldT>
FieldT SHA512_rng(const uint64_t idx)
{
    assert(GMP_NUMB_BITS == 64); // current Python code cannot handle larger values, so testing here for some assumptions.
    assert(is_little_endian());
```

But it's not clear whether this is called outside test code (and knapsack_gadget which I think we don't use).